test/worker-runner.js

```diff
@@ -22,7 +22,7 @@ self.onmessage = function (e) {
     "level1/html": require("../test/level1/html"), // ok
     "level2/core": require("../test/level2/core"), // ok
     "level2/html": require("../test/level2/html"), // 6/708
```

Comment:
Tried it, but the test run crashes without finishing. Haven't looked that much into it yet. We still need to rework the browser test runner somewhat.